app/views/foods_reciipes/general_shopping_list.html.erb

```diff
@@ -0,0 +1,17 @@
+<section class="shoppingItemContainer">
+
+<div class="shopSubHeading">
+<h3>Amount of food item to buy: <%=@shopping_list.size%></h3>
+<h3>Total value of Food needed: $<%=@total%></h3>
+
+</div>
+<div class="foodTable">
+<%= render partial:"foods_reciipes/foods_header"%>
+<% if (@shopping_list.size>0) %>
+<%@shopping_list.each do |recipe_item, index|%>
+<%= render partial:"foods_reciipes/food_recipe_item", locals: {food_item:recipe_item} %>
+<% end %>
+<% else %>
+<% end %>
+</div>
+</section>
```

Comment:
- [ ] Similarly, let's ensure the design of the shopping list matches the given shopping list wireframe design
- [ ] Should show the list of food that is missing for all recipes of the logged-in user (compare the list of food for all recipes with the general food list of that user)
- [ ] Should count the total food items and total price of the missing food.

**Your design:**        |  **Expected:**
:-------------------------:|:-------------------------:
<img width="800" alt="Home Page" src="https://github.com/MayPyone/Recipe-app/assets/12745474/ed0e6f2e-3bfd-4dd5-8688-76dee8ff80a3"> | <img width="800" alt="Home Page" src="https://github.com/MayPyone/Recipe-app/assets/12745474/b2a2f41b-4ccc-4773-b668-b40b6c23c02f">